save-demo/src/main/kotlin/com/saveourtool/save/demo/runners/DefaultRunnerFactory.kt

```diff
@@ -17,11 +17,11 @@ import org.springframework.stereotype.Component
 class DefaultRunnerFactory(
     private val dependencyStorage: DependencyStorage,
     private val commandBuilder: CommandBuilder,
-    private val kubernetesService: KubernetesService,
+    private val kubernetesService: KubernetesService?,
```

Comment:
does spring support it? maybe need to add something like `@Autowired(required = false)`

Reply:
Works fine when running outside Minikube, haven't test inside Kubernetes yet
Will test it

Reply:
It seem to work fine as I use methods from `DemoSerivce` that require `KubernetesSerivce` check